polar_route/vessel_performance/vessels/abstract_glider.py

```diff
@@ -52,7 +52,7 @@ def model_accessibility(self, cellbox):
         access_values = dict()
 
         # Exclude cells due to land or ice
-        access_values['land'] = self.land(cellbox)
+        access_values['land']    = self.land(cellbox)
```

Comment:
Not sure why this needed to be changed, VS code auto-formatting? PEP8 recommends one space either side of assignment operators: https://peps.python.org/pep-0008/#pet-peeves